src/particles/transformation/CoordinateTransformation.cpp

```diff
@@ -75,9 +74,10 @@ namespace transformation {
                     BL_PROFILE("impactx::transformation::CoordinateTransformation::to_fixed_t");
                     amrex::ParticleReal const ptd = pd;  // Design value of pt/mc2 = -gamma.
                     ToFixedT const to_t(ptd);
+                    auto tile_data = pti.GetParticleTile().getParticleTileData();
```

Comment:
This probably needs to be a reference.

But this probably does not copy to GPU, does it?
```suggestion
                    auto & tile_data = pti.GetParticleTile().getParticleTileData();
```